RELEASE.md

```diff
@@ -1,3 +1,83 @@
+# Release 1.15.0
+
+## Major Features and Improvements
+
+## Breaking Changes
+
```

Comment:
Add here (@gunan to confirm words are correct): 

>TensorFlow 1.15 is built using devtoolset7 on Ubuntu 16. This may lead to ABI incompatibilities with extensions built against earlier versions of TensorFlow.

Reply:
Updated. @gunan PTAL